infrastructure/terraform/to_review/notify-service.tf

```diff
@@ -1,131 +0,0 @@
-resource "aws_sqs_queue" "notify_irc_queue" {
-  name = "${var.prefix}-notify-irc"
-}
-
-locals {
-  email_source_address = "${element(split("identity/", var.notify_ses_arn), 1)}"
```

Comment:
We should email_source_address to a terraform variable we pass in to the module since it is used both in tf and in the app.

Add both it and notify_ses_arn to https://github.com/sciurus/taskcluster/blob/cloudops/infrastructure/terraform/variables.tf . You can then delete notify_ses_arn from https://github.com/sciurus/taskcluster/blob/cloudops/infrastructure/terraform/to_review/more_variables.tf#L52

Reply:
Actually nevermind, you just need email_source_address.